commit 128dd32f1d495ebed75b10118424a6688f8de3f0
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Sat Oct 21 00:51:03 2017 +0800

    Launch deepin-notifications on session start
    
    This fixes installations with multiple notification agent
    implementations installed.
    
    Change-Id: I447dcd775a96f890d93351a345faff40b786655a

diff --git a/session.go b/session.go
index e3845c6..fb78c53 100644
--- a/session.go
+++ b/session.go
@@ -274,7 +274,7 @@ func startSession(xu *xgbutil.XUtil) {
 	startStartManager(xu)
 
 	var wg sync.WaitGroup
-	wg.Add(2)
+	wg.Add(3)
 
 	go func() {
 		manager.launch("/usr/bin/dde-desktop", true)
@@ -292,6 +292,11 @@ func startSession(xu *xgbutil.XUtil) {
 		wg.Done()
 	}()
 
+	go func() {
+		manager.launch("/usr/lib/deepin-notifications/deepin-notifications", false)
+		wg.Done()
+	}()
+
 	wg.Wait()
 
 	go func() {
